lib/syscontrol/syscontrol.go

```diff
@@ -225,6 +235,14 @@ func ProcessRequest(req netstorage.SysCtrlRequest, resp *strings.Builder) (err e
 		executor.SetEnableForceBroadcastQuery(enabled)
 		res := "\n\tsuccess"
 		resp.WriteString(res)
+	case TimeFilterProtection:
+		switchon, err := GetBoolValue(req.Param(), "switchon")
```

Comment:
It's a good practice to add some test cases.

Reply:
> It's a good practice to add some test cases.

ok